Analysis/Tasks/PWGHF/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 # granted to it by virtue of its status as an Intergovernmental Organization or
 # submit itself to any jurisdiction.
 
-o2_add_dpl_workflow(qatask
+o2_add_dpl_workflow(qa-event-track
                     SOURCES qaTask.cxx
```

Comment:
It would be better if the file name was more descriptive about the content as well.

Reply:
What would you suggest?

Reply:
Also, don't focus on the task name too much, I plan to have it move elsewhere and merged with other QA tasks, so you can consider letting it go for now